corehq/apps/users/tests/test_create_mobile_workers.py

```diff
@@ -47,11 +47,27 @@ def test_create_unprovisioned(self):
             is_provisioned=False,
         )
         self.addCleanup(user.delete)
-        self.assertEqual(self.domain, user.domain)
         self.assertEqual(False, user.is_active)
         self.assertEqual(False, user.is_provisioned)
         # confirm user can't login
 
         django_user = user.get_django_user()
         self.assertEqual(False, django_user.is_active)
         self.assertEqual(False, self.client.login(username='mw1', password='s3cr4t'))
+
+    def test_is_active_overrides_is_provisioned(self):
```

Comment:
What would you think of putting an assertion inside of `create` to verify `is_provisioned or not is_active`?

Reply:
You're essentially saying to disallow this case and update the test accordingly, yeah?

That makes sense to me. Will change.

Reply:
https://github.com/dimagi/commcare-hq/pull/26864/commits/fc0e6647a64dca27db8ad2bde489e41db299080b